todolist/settings.py

```diff
@@ -62,20 +62,15 @@
 # https://docs.djangoproject.com/en/1.7/ref/settings/#databases
 
 DATABASES = {
-    "default": {
-        "ENGINE": "django.db.backends.sqlite3",
-        "NAME": os.path.join(BASE_DIR, "db.sqlite3"),
-    }
-
-    # "default": {
-    #     "ENGINE": "django.db.backends.mysql",
-    #     "HOST": "FVFFM3F5Q05N", 
-    #     "PORT": 3306,
-    #     "NAME":"tododb",
-    #     "PASSWORD": "my-secret-pw", 
-    #     "USER": "root",
-    # }
-}
+      'default': {
+          'ENGINE': 'mysql.connector.django',
+          'NAME': 'app_db',
+          'USER': 'app_user',
+          'PASSWORD': '1234',
```

Comment:
The database PASSWORD is hardcoded, which is a security risk. Consider using environment variables or a secure vault to manage sensitive information.